loader/loader_linux.c

```diff
@@ -270,7 +270,6 @@ VkResult linux_read_sorted_physical_devices(struct loader_instance *inst, uint32
 
             bool device_is_1_1_capable =
                 VK_API_VERSION_MAJOR(dev_props.apiVersion) == 1 && VK_API_VERSION_MINOR(dev_props.apiVersion) >= 1;
-            sorted_device_info[index].has_pci_bus_info = device_is_1_1_capable;
```

Comment:
How does has_pci_bus_info get set now? it looks like its always initialized to false.

Reply:
Look at line 289.